src/core/app/init.ts

```diff
@@ -41,9 +40,6 @@ export async function appInit() {
 
   updateAppConfig(appInfo);
 
-  // FIXME: refactor remove currentApplicationAtom dependency in other atoms
-  currentApplicationAtom.set.dispatch(initialState.app);
```

Comment:
currentApplicationAtom still stays in src/features/create_layer/readme.md
need to remove from docs as it doesn't exist

Reply:
good point